From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

Commit:
more tweaks

ruoxinCheng committed:
Update hdi vscode doc and add a new setup environment doc

ruoxinCheng committed:
Update doc again

ruoxinCheng committed:
update the third time

Commit:
Updates per PM.

Commit:
Fix

ruoxinCheng committed:
update long comments

ruoxinCheng committed:
fix format

ruoxinCheng committed:
add TOC

Rajani-Janaki-Ram committed:
edit image

v-geberr committed:
simplify about page

v-geberr committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into 171027-aboutLUIS

v-geberr committed:
add info about keys

rboucher committed:
OMS moniker is going away. 

Silently removing references.

Rajani-Janaki-Ram committed:
editing iaas

Rajani-Janaki-Ram committed:
editing iaas

MGoedtel committed:
new manage articles

MGoedtel committed:
updated cost article

MGoedtel committed:
updated cost article

genlin committed:
update the toc

ruoxinCheng committed:
Move install pathon ext to pySpark interactive and add some links in TOC

genlin committed:
add the file into redirection file

genlin committed:
revert the redirection filke

genlin committed:
add file

genlin committed:
add redirection files

genlin committed:
remove the three billing articles

ruoxinCheng committed:
fixed format and typo

ruoxinCheng committed:
Change a screenshot

genlin committed:
fix the warning problem

ruoxinCheng committed:
add content

v-geberr committed:
rewrite and better image

v-geberr committed:
fixed link, rewrote next steps

mmacy committed:
[acr] docker login security note

v-geberr committed:
changes from KI and DM review

yujhongmicrosoft committed:
Create documentation-government-GPU.md

yujhongmicrosoft committed:
Rename documentation-government-GPU.md to documentation-government-gpu.md

yujhongmicrosoft committed:
Update TOC.md

cherylmc committed:
note

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into test

spelluru committed:
new PS sample - Bulk Copy

yujhongmicrosoft committed:
Update documentation-government-gpu.md

spelluru committed:
first draft of incremental copy sample

yujhongmicrosoft committed:
Update documentation-government-gpu.md

spelluru committed:
broken link errors

spelluru committed:
fixed broken link

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into bulkcopysample1031

spelluru committed:
few corrections

kgremban committed:
Consolidated Create IoT Hub includes

mimig1 committed:
Cosmos DB: Corrected meta.

spelluru committed:
added next steps after script

cherylmc committed:
new

KumudD committed:
updated meta data

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into DNSAuthorNameUpdate

spelluru committed:
fixed minor issues

cherylmc committed:
toc

kgremban committed:
Formatting

cherylmc committed:
update

KumudD committed:
cross-linked private dns topics

barclayn committed:
making a minor change per David

cherylmc committed:
Update expressroute-gateway-classic-ps-include.md

v-geberr committed:
applied KI suggestions

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ermonitor

kgremban committed:
Formatting

Commit:
Merge pull request #26621 from barclayn/genericsql

making a minor change per David

v-geberr committed:
hard link to Speech/index.md

barclayn committed:
fixing metadata formatting issue

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ermonitor

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into bulkcopysample1031

Lisaco88 committed:
Merge pull request #26600 from cherylmc/test

note for coexist

spelluru committed:
class portal references

PRMerger19 committed:
Merge pull request #25666 from JennieHubbard/patch-94

more tweaks

Commit:
catechref01

Commit:
Merge pull request #26624 from barclayn/azure-encryption

fixing metadata formatting issue

PRMerger12 committed:
Merge pull request #26445 from rboucher/patch-7

OMS moniker is going away.

v-thepet committed:
fix

Commit:
update steps

reediting because this PR: https://github.com/MicrosoftDocs/azure-docs-pr/pull/26379 was merged, but never made it to master?

PRMerger13 committed:
Merge pull request #26627 from v-thepet/fixy

production bug: fix

Lisaco88 committed:
Merge pull request #26608 from kgremban/oct31_hubinclude

Consolidated Create IoT Hub includes

Commit:
Merge pull request #26625 from MarkusVi/catechref01

catechref01

markgalioto committed:
changing author attribute for Backup articles

MGoedtel committed:
updated articles

PRMerger16 committed:
Merge pull request #26632 from spelluru/classic1031

class portal references

mrbullwinkle committed:
Update ms.author

spelluru committed:
updated powershell samples main page

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb_10312017B

MGoedtel committed:
corrected errors

PRMerger15 committed:
Merge pull request #26630 from stevestein/patch-4

update steps

spelluru committed:
cleanup
:

mrbullwinkle committed:
acrolinx issue

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb_10312017B

ktoliver committed:
Merge pull request #26478 from MGoedtel/NewManageArticles10302017

new manage articles

ktoliver committed:
Merge pull request #26564 from genlin/ci68377

Redirect three billing articles

ktoliver committed:
Merge pull request #26634 from mrbullwinkle/mrb_10312017B

Mrb 10312017 b

ruoxinCheng committed:
Resolved blocing issues

ruoxinCheng committed:
fixed format

PRMerger17 committed:
Merge pull request #26631 from markgalioto/AB-10312017

changing author attribute for Backup articles

ruoxinCheng committed:
Add links to next steps

ruoxinCheng committed:
fixed format

dlepow committed:
Batch: compute-intensive sizes updates

dlepow committed:
edit

ponatara committed:
Update site-recovery-support-matrix-to-azure.md

Rajani-Janaki-Ram committed:
namechange

Rajani-Janaki-Ram committed:
namechange

Rajani-Janaki-Ram committed:
namechange

PRMerger18 committed:
Merge pull request #26641 from ponatara/master

Update site-recovery-support-matrix-to-azure.md

Rajani-Janaki-Ram committed:
namechange

linda33wj committed:
Add OID auth support for Oracle

linda33wj committed:
Add tips for ADLS SP

linda33wj committed:
Add MySQL driver tips

linda33wj committed:
Fix broken link and typo

mrbullwinkle committed:
ms.custom tag added to overview articles for reporting

PRMerger14 committed:
Merge pull request #26645 from linda33wj/master

Update ADF copy content

mrbullwinkle committed:
build error change to force rebuild

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb_11012017

Commit:
Merge pull request #26646 from mrbullwinkle/mrb_11012017

ms.custom tag added to overview articles for reporting

PRMerger12 committed:
Merge pull request #26064 from jimdial/patch-14

Updates per PM.

v-geberr committed:
removed flow

rayne-wiselman committed:
adding MVC tag to docs that went thru the ignite MVC process

daveirwin1 committed:
Added Southeast Asia region

Commit:
Merge pull request #26581 from mmacy/acr-docker-login-fix

[acr] docker login security note

tfitzmac committed:
revised powershell tag example

Lisaco88 committed:
Merge pull request #26601 from spelluru/bulkcopysample1031

new PS sample - Bulk Copy

cherylmc committed:
updates

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ermonitor

PRMerger10 committed:
Merge pull request #26655 from daveirwin1/master

Added Southeast Asia region

Lisaco88 committed:
Merge pull request #26060 from ruoxinCheng/vscode

Update hdi vscode doc and add a new setup environment doc

Commit:
Merge pull request #26618 from KumudD/DNSAuthorNameUpdate

Cross-link to private DNS topics

PRMerger15 committed:
Merge pull request #26611 from mimig1/patch-152

Cosmos DB: Corrected meta.

Lisaco88 committed:
Merge pull request #26653 from rayne-wiselman/rayne-november1

adding MVC tag to docs that went thru the ignite MVC process

twooley committed:
added PowerShell syntax difference

PRMerger19 committed:
Merge pull request #26661 from twooley/twrelnotesyntax

added PowerShell syntax difference

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger10 committed:
Merge pull request #26639 from dlepow/fix2

Batch: compute-intensive sizes updates

Lisaco88 committed:
Merge pull request #26613 from cherylmc/ermonitor

new article

Lisaco88 committed:
Merge pull request #26598 from yujhongmicrosoft/patch-2

Create documentation-government-GPU.md

rjagiewich committed:
Merge pull request #26433 from v-geberr/171027-aboutLUIS

171027 about luis

Lisaco88 committed:
Merge pull request #26657 from tfitzmac/1101tag

revised powershell tag example

v-alje committed:
Merge pull request #26662 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #26383 from Rajani-Janaki-Ram/master

Master
